why-evolveum/get-value-quickly-with-iterative-approach.adoc

```diff
@@ -0,0 +1,23 @@
+= Get Value Quickly with Evolveum's Iterative Approach to IGA
+:page-nav-title: 'Advantages of Iterative Approach'
+:page-display-order: 2
```

Comment:
Typically you want to use larger numbers, like 10, 20, ... or 100, 200, ... 
Then, you can insert a page in between without reordering the rest.